### PR TITLE
OpenTelemetry Collector for Honeycomb

### PR DESCRIPTION
**Pull request by puckpuck**
chart to deploy the OpenTelemetry collector configured with recommended defaults and the Honeycomb exporter

### CODE REVIEW

File: opentelemetry-collector/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+apiVersion: v2
+name: opentelemetry-collector
+description: OpenTelemetry Collector for Honeycomb
+type: application
+version: 0.1.0
+appVersion: 0.4.0
+keywords:
+  - observability
+  - tracing
+home: https://honeycomb.io
```
Comment (irvingpop) [inline opentelemetry-collector/Chart.yaml:10]: is this the right homepage?

File: opentelemetry-collector/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+sources:
+  - https://github.com/honeycombio/opentelemetry-collector
+  - https://github.com/honeycombio/opentelemetry-collector-contrib
+icon: https://www.honeycomb.io/wp-content/themes/honeycomb/assets/img/logo.svg
```
Comment (irvingpop) [inline opentelemetry-collector/Chart.yaml:14]: probably should be the OTel logo, right?

File: opentelemetry-collector/templates/NOTES.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{{- $validApiKey := and (.Values.honeycomb.apiKey) (ne .Values.honeycomb.apiKey "YOUR_API_KEY") -}}
+
+{{- if not $validApiKey }}
+##### ERROR:
```
Comment (irvingpop) [inline opentelemetry-collector/templates/NOTES.txt:4]: naive question, what happens if you hit this error condition? 

also if this is helpful/obvious to users, should we also require dataset be set?

**Comment by irvingpop** [comment]
few minor things, other than that this looks great!

File: opentelemetry-collector/templates/NOTES.txt (hunk 3 above)
Comment (puckpuck) [reply opentelemetry-collector/templates/NOTES.txt:4]: dataset has a default name, and the Readme does explain how to change it, though we may want to make that part more prominent. If you don't specify an API key, the collector will throw unauthorized errors (via Honeycomb exporter).

**Pull request merged**